### PR TITLE
feat: add query suggestions

### PR DESCRIPTION
**Pull request by tiborux**
EX-6305

### CODE REVIEW

**Review by diegopf — APPROVED** [review]

**Review by mnavarroespinosa — COMMENTED** [review]
The export for the `query-suggestions-request.mapper` is missing in `packages/x-adapter-platform/src/mappers/requests/index.ts` file. And the export for the response mapper is also missing.

Could you add the `index.ts` file with the exports for `schemas/models` directory?

**Comment by tiborux** [comment]
> The export for the `query-suggestions-request.mapper` is missing in `packages/x-adapter-platform/src/mappers/requests/index.ts` file. And the export for the response mapper is also missing.
> 
> Could you add the `index.ts` files with the exports for `schemas/models` and `types/models` directories?

done

File: packages/x-adapter-platform/src/mappers/responses/empathize-response.mapper.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { createMutableSchema, Schema, schemaMapperFactory } from '@empathyco/x-adapter-next';
 import { EmpathizeResponse, PlatformEmpathizeResponse } from '../../types/response.types';
-import { suggestionMutableSchema } from '../../schemas/suggestion.schema';
+import { suggestionMutableSchema } from '../../schemas';
```
Comment (mnavarroespinosa) [inline packages/x-adapter-platform/src/mappers/responses/empathize-response.mapper.ts:3]:
```suggestion
import { suggestionMutableSchema } from '../../schemas/models/suggestion.schema';
```

**Comment by mnavarroespinosa** [comment]
gj!

**Review by mnavarroespinosa — APPROVED** [review]

**Review by diegopf — APPROVED** [review]

**Pull request merged**